src/base-card/style.js

```diff
@@ -26,12 +26,13 @@ const HardThickShadow = css`
   box-shadow: 10px 25px 30px 0 rgba(7, 0, 37, 0.4);
 `;
 
-export default styled.div`
+export default styled.a`
```

Comment:
Do we want all our cards to be anchors?
Aren't we using these card components in Valuation Wizard?

Reply:
Those card components in val wizard have been made in val wizard (they don't use oxygen).

I did raise the question but I can't think of a case where they wouldn't be an anchor? If so then we have to explore `as` a little more and make on PropertyCard an anchor.

Reply:
Okay cool, I'm sure we can swap it out somewhere down the line if it starts being an issue.